lib/ansible/modules/cloud/amazon/iam_user_info.py

```diff
@@ -175,7 +172,7 @@ def main():
 
     region, ec2_url, aws_connect_params = get_aws_connection_info(module, boto3=True)
```

Comment:
You can delete this row too now

Reply:
https://github.com/ansible/ansible/pull/23382/commits/450fd12d0898960b65177a0d10f14765fa39d5fc removes this.

Reply:
It doesn't

Reply:
Oops.. https://github.com/ansible/ansible/pull/23382/commits/34cc91a8d2b54d6436bda2c744f777fdc9600ebc corrects this now.